common.mk

```diff
@@ -40,7 +40,7 @@ REF                ?= HEAD
 DOCKER_CLI_REF     ?= $(REF)
 DOCKER_ENGINE_REF  ?= $(REF)
 DOCKER_SCAN_REF    ?= v0.17.0
-DOCKER_COMPOSE_REF ?= v2.10.0
+DOCKER_COMPOSE_REF ?= 9c23e015228d68bf1038343781d8a84283a9a545
```

Comment:
Went for commit to be sure (at risk that it changes if you did a force-push), but 😂